From: Roland Hieber <rhi@pengutronix.de>
Date: Mon, 24 May 2021 22:52:00 +0200
Subject: [PATCH] lib/usleep.c: fix error: 'NULL' undeclared

This fixes compilation with GCC 11:

    usleep.c: In function 'usleep':
    usleep.c:14:12: error: 'NULL' undeclared (first use in this function)
    usleep.c:14:12: note: 'NULL' is defined in header '<stddef.h>'; did you forget to '#include <stddef.h>'?
    usleep.c:14:12: note: each undeclared identifier is reported only once for each function it appears in
    make[2]: *** [Makefile:365: usleep.o] Error 1
---
 lib/usleep.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/usleep.c b/lib/usleep.c
index af9dd0326024..861c00ad97d1 100644
--- a/lib/usleep.c
+++ b/lib/usleep.c
@@ -2,6 +2,7 @@
 # include <config.h>
 #endif
 
+#include <stddef.h>
 #include <sys/time.h>
 #include <sys/select.h>
 
